src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/DefaultEditorTemplates.cs

```diff
@@ -350,7 +350,7 @@ public static IHtmlContent EmailAddressInputTemplate(IHtmlHelper htmlHelper)
         public static IHtmlContent DateTimeInputTemplate(IHtmlHelper htmlHelper)
         {
             ApplyRfc3339DateFormattingIfNeeded(htmlHelper, "{0:yyyy-MM-ddTHH:mm:ss.fffK}");
-            return GenerateTextBox(htmlHelper, inputType: "datetime");
+            return GenerateTextBox(htmlHelper, inputType: "datetime-local");
```

Comment:
Ah, I thought I was toeing closer to the @Eilon line. See [other thread](https://github.com/aspnet/Mvc/pull/5664#discussion_r95442995) on the subject.